interfaces/acados_template/acados_template/acados_multiphase_ocp.py

```diff
@@ -194,6 +195,23 @@ def parameter_values(self, parameter_values):
             raise Exception('parameter_values must be a list of length n_phases.')
         self.__parameter_values = parameter_values
 
+
+    @property
+    def p_global_values(self):
+        """:math:`p` - list of initial values for p_global vector.
+        Type: `list` of `numpy.ndarray` of shape `(np_global_i, )`.
+        - can be updated stagewise."""
+        return self.__p_global_values
+
+    @p_global_values.setter
+    def p_global_values(self, p_global_values):
+        if not isinstance(p_global_values, list):
+            raise Exception('p_global_values must be a list of numpy.ndarrays.')
+        elif len(p_global_values) != self.n_phases:
+            raise Exception('p_global_values must be a list of length n_phases.')
+        self.__p_global_values = p_global_values
```

Comment:
`p_global` should be a single numpy array also for MOCP.
I remember that we opted for an implementation where `p_global` has to be the same vector for all stages (including MOCP formulation) and that if one indeed wants different parameters for the phases they all need to be in `p_global`.
Entries of `p_global` can be used across multiple phases.
This is actually already checked here:
https://github.com/acados/acados/blob/master/interfaces/acados_template/acados_template/acados_multiphase_ocp.py#L246-L252